docs/src/main/paradox/slick.md

```diff
@@ -125,3 +125,5 @@ Scala
 
 Java
 : @@snip ($alpakka$/slick/src/test/java/akka/stream/alpakka/slick/javadsl/DocSnippetFlow.java) { #flow-example }
+
+ [jdbcbackend-api]: http://slick.lightbend.com/doc/3.2.1/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver,ClassLoader):Database
```

Comment:
You could define Slick Version in `Dependencies` and use it to keep it pointing to the version in use.
Looks like more API links are version dependent...

Reply:
That doesn't appear to work inside URL's (https://github.com/akka/alpakka/tree/slickVersionFromDependencies)